plugins/org.eclipse.n4js/src/org/eclipse/n4js/internal/N4JSModel.java

```diff
@@ -442,6 +442,17 @@ public String getDefinesPackage(final IN4JSProject project) {
 		return null;
 	}
 
+	public boolean isUseES6Imports(final IN4JSProject project) {
+		if (project == null) {
+			return false;
+		}
+		final ProjectDescription projectDescription = getProjectDescription(project.getLocation());
+		if (projectDescription != null) {
```

Comment:
minor but to be consistent 

```
if (projectDescription == null) {
   return null;
}
```

?

Reply:
I had aligned that to the method right above this one.